protoactor/protos_pb2.py

```diff
@@ -0,0 +1,267 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
However if you find the way how to not include this file. It would cool to know.